.travis.yml

```diff
@@ -19,6 +19,8 @@ before_install:
   # There is a bug in travis. When using system ruby, bundler is not
   # installed and causes the default install action to fail.
   - sudo gem install bundler
+  - bundle install
+  - bundle exec pod repo update
 
 after_install:
   - ./bin/pod repo update --silent
```

Comment:
ok, so after reading the travis docs, I think we can back out the changes to `before_install`, and change this section to

```yaml
before_script:
  - bundle exec pod repo update --silent
```